README.md

```diff
@@ -1,25 +1,50 @@
 # moodle-mod_observation
 
-Create and manage observation assessments. This plugin is primarily designed 
-for educational facilities as an additional method to conduct assessment tasks 
-in courses requiring lots of practical work such as nursing and chemistry, among others.
+This plugin allows easy management of observational assessments. 
 
-## Development Version
-Warning! Never use the development version in production, there are no guarantees for which state the development branches are in at a given time.
+These assessment usually involve an 'observer' (such as a teacher, tutor, supervisor, etc) and an 'observee' (student, employee, etc). The observer watches the observee as they do a process (such as pour drinks at a bar or play piano), recording if they completed certain 'observation points' during the process. 
 
-## Installation
+These 'observation points' could be key assessment criteria, such as health and safety requirements or specific processes the observee must do. Currently three different types are supported: Pass/fail, Text feedback, and File upload (e.g. images). Observers can record a response and assign a mark for each point.
+
+After an observation session is complete, marks and feedback are stored in the Moodle gradebook for review.
+
+This plugin also supported a timeslot management system, allowing timeslots to be created, assigned to observers and joined by observees - notifications included.
+
+## Branches
+| Moodle Version      | Branch |
+| ----------- | ----------- |
+| 3.5+      | master       |
```

Comment:
This should be `main` not `master`.....But just putting it out there for discussion, should we follow our standard as we do for other plugins?

i.e not have a "main" branch, but a `MOODLE_XX_STABLE` naming convention?

Reply:
Ahh copy paste error - should definitely by `main` not `master`.

In response to the other naming convention - I think it might be a bit redundant as this plugin doesn't really use any special Moodle features that would require very specific version controls. Perhaps 4.0 will break it but thats another story altogether...

Reply:
Yeh, I was just thinking ahead for 4.0....but i think its safe to merge it in as it for now..